### ISSUE TITLE
BinaryNinja infinite analysis on PE

### ISSUE THREAD

**Issue by krornus** [Vector35/binaryninja-api#1208]
Issue: BinaryNinja infinitely analyzes dui70.dll from system32 (64bit). The file is 1.7Mb and has so far been analyzing for a couple days. It is still in stage one.
[dui70.zip](https://github.com/Vector35/binaryninja-api/files/2626844/dui70.zip)

**Comment by joshwatson** [comment]
According to the analysis info, it's gotten stuck in a loop analyzing the function at 0x1800d3ad0:

```
<AnalysisInfo AnalysisState.AnalyzeState, analysis_time 196484, active_info [<ActiveAnalysisInfo <func: x86_64@0x1800d2c40>, analysis_time 19, update_count 0, submit_count 4644>, <ActiveAnalysisInfo <func: x86_64@0x1800d3ad0>, analysis_time 140680, update_count 5573, submit_count 1>]>
```

**Comment by bpotchik** [comment]
Thanks Josh. A way to temporarily side-step an issue like this is to run the analysis_info command as shown above and then undefine the offending function. Analysis did finish for me once I did that. I may end up adding a setting which can auto-defer a specific function that exceeds a max update_count.

The real fix which allows analysis of the offending function to finish will show up shortly.

**Comment by bpotchik** [comment]
A new setting (analysis.maxFunctionUpdateCount) was added to build 1.1.1440-dev. This allows an excessive function update cycle to be automatically terminated and a warning generated. Please still report these issues though.

Leaving open until the real issue is fixed.

**Comment by bpotchik** [comment]
Underlying issue fixed in 1.1.1442-dev.
